src/compute/Cargo.toml

```diff
@@ -28,7 +28,7 @@ mz-dyncfg = { path = "../dyncfg" }
 mz-dyncfgs = { path = "../dyncfgs" }
 mz-expr = { path = "../expr" }
 mz-metrics = { path = "../metrics" }
-mz-ore = { path = "../ore", features = ["async", "process", "tracing", "columnar", "differential-dataflow"] }
+mz-ore = { path = "../ore", features = ["async", "process", "tracing", "columnar", "differential-dataflow", "region"] }
```

Comment:
What is this about, btw?

Reply:
A missing feature, should have been there but I forgot to add it. It compiles because some dependency enables it, too, but we shouldn't rely on this behavior.